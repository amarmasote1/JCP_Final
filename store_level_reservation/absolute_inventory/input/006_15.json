{
	"reservation": [{
		"storeID": "1056",
		"itemID": "35145678012",
		"uom": "EACH",
		"shippable": "true",
		"procurementAllowed": "false",
		"qty": "0",
		"lineID": "1",
		"reservationID": "R36003",
		"increaseCapacity": "Y"
	}]
}