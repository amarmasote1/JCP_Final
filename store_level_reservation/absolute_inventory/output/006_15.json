{
    "message": "Success",
    "body": [
        {
            "partiallyReservedQty": 0,
            "shortageQty": 0,
            "orgCode": "JCP",
            "reservedQty": 0,
            "requestLine": {
                "deliveryMethod": null,
                "channel": null,
                "lineID": "1",
                "storeID": "1056",
                "itemID": "3514567-8012",
                "schedule": false,
                "uom": "EACH",
                "shippable": true,
                "reservationID": "R36003",
                "procurementAllowed": false,
                "qty": 0,
                "bopus": false,
                "increaseCapacity": "Y",
                "shutdown": false
            },
            "problemType": "SUCCESS"
        }
    ],
    "status": 200
}