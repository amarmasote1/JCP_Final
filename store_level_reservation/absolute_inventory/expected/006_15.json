{
	"status": 200,
	"message": "Success",
	"body": [{
		"shortageQty": 0,
		"reservedQty": 0,
		"partiallyReservedQty": 0,
		"problemType": "SUCCESS",
		"orgCode": "JCP",
		"requestLine": {
			"lineID": "1",
			"itemID": "3514567-8012",
			"uom": "EACH",
			"bopus": false,
			"shippable": true,
			"deliveryMethod": null,
			"increaseCapacity": "Y",
			"shutdown": false,
			"qty": 0,
			"storeID": "1056",
			"channel": null,
			"reservationID": "R36003",
			"procurementAllowed": false
		}
	}]
}